arnold/__init__.py

```diff
@@ -1,164 +1,242 @@
 import os
+import sys
+
+import argparse
 
 from termcolor import colored
 
-from arnold.exceptions import (
-    ArgumentException,
-    DBAttrNotFound,
-    DirectionNotFoundException,
-    InvalidConfiguration,
-    ModuleNotFoundException,
-    MigrationNotFoundException,
-)
+from arnold.exceptions import DirectionNotFoundException
 from arnold.models import Migration
 from importlib import import_module
 
-import logging
-logger = logging.getLogger(__name__)
-
-
-IGNORED_FILES = ["__init__"]
 
+class Terminator:
+    IGNORED_FILES = ["__init__"]
 
-def _setup_table(model):
-    if model.table_exists():
-        return
-    else:
-        model.create_table()
-    return True
+    def __init__(self, args):
```

Comment:
Instead of providing an `optparse` instance which is hard to use during tests you could provide a dictionary. (Ex.: `args.__dict__`).

Worth mentioning that `args` is not a very good name, for example `pdb.set_trace()` will shadow it.